Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -589,6 +589,60 @@ private IEnumerator TestTouchableDistances(BaseNearInteractionTouchable touchabl
 
             yield return PlayModeTestUtilities.HideHand(handedness, inputSim);
         }
+
+
+        /// <summary>
+        /// Test the SetTouchableCollider(BoxCollider collider) method by changing the size of the new
+        /// box collider and rotating the object. This test is checking if the NearInteractionTouchable plane
+        /// is the same size as the box collider and in front of the object.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator NearInteractionTouchableSetTouchableCollider()
```

Comment:
I currently have a test moving a hand and checking isNearObject through the poke pointer. 
@julenka  What are your thoughts?

```
        [UnityTest]
        public IEnumerator NearInteractionTouchableSetTouchableCollider()
        {
            GameObject cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
            cube.transform.position = new Vector3(0, 0, 2f);
            cube.transform.localScale = new Vector3(0.3f, 0.3f, 0.3f);

            var rightHand = new TestHand(Handedness.Right);
            yield return rightHand.Show(new Vector3(0, 0, 1.5f));
            var pokePointer = PointerUtils.GetPointer<PokePointer>(Handedness.Right);
            Assert.IsNotNull(pokePointer);

            // Test isNearObject without NearInteractionTouchable
            Assert.False(pokePointer.IsNearObject);

            yield return rightHand.Move(new Vector3(0, 0, 0.2f));

            Assert.False(pokePointer.IsNearObject);

            // Move the hand back
            yield return rightHand.Move(new Vector3(0, 0, -0.2f));

            // Test isNearObject with NearInteractionTouchable
            var nearIT = cube.AddComponent<NearInteractionTouchable>();
            var boxCollider = cube.GetComponent<BoxCollider>();

            nearIT.SetTouchableCollider(boxCollider);

            Assert.False(pokePointer.IsNearObject);

            yield return rightHand.Move(new Vector3(0, 0, 0.2f));

            Assert.True(pokePointer.IsNearObject);
        }
```

Reply:
This test does call nearIT.SetTouchableCollider() but it does not need to because this was added to NearInteractionTouchable:

```
        public void OnEnable()
        {
            if (touchableCollider == null)
            {
                SetTouchableCollider(GetComponent<BoxCollider>());
            }
        }
```

Reply:
agreeing with Julia, you would want to test the behavior of that touchable when you're interacting with it. I wouldn't do any checks on the pokepointer when the nearinteractionTouchable is attached or not. Testing the IsNearObject shouldn't be part of your test. That can be another test, but this one should focus on testing your new method in the NearInteractionTouchable. 
Also it does make sense to not only test the default behavior you mentioned, but also to manually set a new collider that isn't the own gameobjects one, to verify that the bounds properly adjust by calling your method. 

Use Julias steps for the test scenario and provide two test cases, one for default behavior , one for setting a collider from a different gameobject that has different bounds and make sure that the interaction works and bounds are calculated / set correctly